int8 num_mon
int8 num_track
int8 available_track